tests/agent/test_tool_cache.py

```
@@ -0,0 +1,91 @@
+        key = cache.store(result)
+        assert cache.retrieve(key) == result
+
+    def test_lru_eviction(self):
+        """超出容量时淘汰最旧"""
+        cache = ToolResultCache(max_size=3)
+        keys = [cache.store({"v": i}) for i in range(5)]
+        assert keys[0] not in cache._cache  # 第一条被淘汰
+        assert cache.size == 3
+
+    def test_grep_summarize(self):
+        """grep 工具：摘要只保留 stats，不保留 matches"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)